desktop/release-notes.md

```diff
@@ -23,17 +23,95 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
+- Implemented feedback on the containers table. Added ports and separated container and image names.
+- Added two new extensions, Ddosify and Lacework, to the Extensions Marketplace.
+
+### Upgrades
+- [Docker Engine to v20.10.17](https://docs.docker.com/engine/release-notes/#201017)
+- [Docker Compose v2.6.1](https://github.com/docker/compose/releases/tag/v2.6.1)
+- [Kubernetes v1.24.1](https://github.com/kubernetes/kubernetes/releases/tag/v1.24.1)
+- [cri-dockerd to v0.2.1](https://github.com/Mirantis/cri-dockerd/releases/tag/v0.2.1)
+- [CNI plugins to v1.1.1](https://github.com/containernetworking/releases/tag/v1.1.1)
+- [containerd to v1.6.6](https://github.com/containerd/containerd/releases/tag/v1.6.6)
+- [runc to v1.1.2](https://github.com/opencontainers/runc/releases/tag/v1.1.2)
+- [Go 1.18.3](https://github.com/golang/go/releases/tag/go1.18.3)
+
+### Bug fixes and minor changes
+
+#### For all platforms
+
+- Added additional bulk actions for starting/pausing/stopping select containers in the **Containers** tab.
+- Added pause and restart actions for compose projects in the **Containers** tab.
+- Added icons and exposed ports or exit code information in the **Containers** tab.
+- External URLs can now refer to extension details in the Extension Marketplace using links such as `docker-desktop://extensions/marketplace?extensionId=docker/logs-explorer-extension`.
+- The expanded or collapsed state of the Compose apps is now persisted.
+- `docker extension` CLI commands are available with Docker Desktop by default.
+- Increased the size of the screenshots displayed in the Extension marketplace.
+- Fixed a bug where a Docker extension fails to load if its backend container(s) are stopped. Fixes [docker/extensions-sdk#16](https://github.com/docker/extensions-sdk/issues/162).
+- Fixed a bug where the image search field is cleared without a reason. Fixes [docker/for-win#12738](https://github.com/docker/for-win/issues/12738).
+- Fixed a bug where the license agreement does not display and silently blocks Docker Desktop startup.
+- Fixed the displayed image and tag for unpublished extensions to actually display the ones from the installed unpublished extension.
+- Fixed the duplicate footer on the Support screen.
+- Dev Environments can be created from a subdirectory in a GitHub repository.
+
+#### For Mac
+
+- Fixed a bug with location of bash completion files on macOS. Fixes [docker/for-mac#6343](https://github.com/docker/for-mac/issues/6343).
+- Fixed a bug where Docker Desktop does not start if the username is longer than 25 characters. Fixes [docker/for-mac#6122](https://github.com/docker/for-mac/issues/6122).
+- Fixed a bug where Docker Desktop was not starting due to invalid system proxy configuration. Fixes some issues reported in [docker/for-mac#6289](https://github.com/docker/for-mac/issues/6289).
+- Fixed a bug where a startup race occurred when the experimental virtualization framework is enabled.
```

Comment:
This is the cause: what was the symptom from the user's pov?

Reply:
I would assume Docker Desktop not starting, @djs55 is that correct ?

Reply:
Symptom was failing to startup (underneath the Swift code started the engine etc, then opened the dashboard, then exited, then everything else would either exit or get stuck depending on timing).

Maybe omit the mention of the race?
```
Fixed a bug where Docker Desktop failed to start when the experimental virtualization.framework is enabled.
```